inventory/.env.sample

```diff
@@ -0,0 +1,13 @@
+SECRET_KEY=
+DEBUG=
```

Comment:
Hi @katendejericho5 , wondering what Inventory changes are doing in a mobile app PR?